### PR TITLE
Added Control panel permissions reference

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: essential-knowledge/cp-permissions.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+| collaborator  |collaborator|              |               |
++---------------+------------+--------------+---------------+
+| Applications  | n/a        |              |               |
+| collaborator  |            |              |               |
++---------------+------------+--------------+---------------+
+|  Organisation |            |              |               |
+|  owner/admin  |            |              |               |
++---------------+------------+--------------+---------------+
+
+.. note:: For migration:
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: This is an internal note for myself.

File: essential-knowledge/cp-permissions.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+| Addon         | n/a         |              | can also      |               |                 |                 |
+| collaborator  |             |              | remove        |               |                 |                 |
+| with "can     |             |              | themselves    |               |                 |                 |
+| update"       |             |              |               |               |                 |                 |
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
+| Organisation  |             |              |               |               |                 |                 |
+| owner/admin   |             |              |               |               |                 |                 |
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
+
+*Access to a project is a prerequisite for this action.
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: RST has dedicated markup for footnotes.

File: essential-knowledge/cp-permissions.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
+.. _knowledge-cp-permissions:
+
+Control panel permissions reference
+====================================
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: Length does not match title.

File: essential-knowledge/cp-permissions.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+The following table shows the different permission options for the different
+types of users with respect to addons, boilerplates and applications.
+
+
+Addons
+----------
+
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
+| User level    |  Create new | Manage addon | Manage addon  | Upload new    | Install/upgrade | Manage settings |
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: Where is the information about what is allowed/denied? We need some sort of indicator.

File: essential-knowledge/cp-permissions.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+====================================
+
+The following table shows the different permission options for the different
+types of users with respect to addons, boilerplates and applications.
+
+
+Addons
+----------
+
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: There is a `..list-table` directive which may be useful for *editing*: https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table

File: essential-knowledge/cp-permissions.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+Control panel permissions reference
+====================================
+
+The following table shows the different permission options for the different
+types of users with respect to addons, boilerplates and applications.
+
+
+Addons
+----------
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: Does not match length.

File: essential-knowledge/cp-permissions.rst [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
+| Organisation  |             |              |               |               |                 |                 |
+| owner/admin   |             |              |               |               |                 |                 |
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
+
+*Access to a project is a prerequisite for this action.
+
+
+Boilerplates
+-------------
```
Comment (GaretJax) [inline essential-knowledge/cp-permissions.rst]: Does not match length.

**Review by GaretJax — CHANGES_REQUESTED** [review]

File: essential-knowledge/cp-permissions.rst (hunk 1 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: I made it internal note.

File: essential-knowledge/cp-permissions.rst (hunk 2 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: Done

File: essential-knowledge/cp-permissions.rst (hunk 3 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: Done

File: essential-knowledge/cp-permissions.rst (hunk 4 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: I need more information on this.

File: essential-knowledge/cp-permissions.rst (hunk 5 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: I have used both styles, the List-table and the Grid-table. I would go for the Grid table, even though tedious to write, it has better features. Let me know your thoughts.

File: essential-knowledge/cp-permissions.rst (hunk 6 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: Done

File: essential-knowledge/cp-permissions.rst (hunk 7 above)
Comment (mebverse) [reply essential-knowledge/cp-permissions.rst]: Done

**Review by vxsx — APPROVED** [review]

**Review by GaretJax — APPROVED** [review]

**Pull request merged**